### ISSUE TITLE
v0.1.7 image removed from ghcr.io  

### ISSUE THREAD

**Issue by Hannessi** [interlynk-io/sbomqs#318]
A new version was recently released. However we have difficulties getting this version up and running on Openshift. We want to use the previous version v0.1.7, but this image has been removed from ghcr.io.

Locally,
```
➜  ~ docker pull ghcr.io/interlynk-io/sbomqs:v0.1.7
v0.1.7: Pulling from interlynk-io/sbomqs
no matching manifest for linux/arm64/v8 in the manifest list entries
```

From the clusters,
```
ERROR: Job failed: prepare environment: waiting for pod running: pulling image "ghcr.io/interlynk-io/sbomqs:v1.7.0": 
image pull failed: rpc error: code = Unknown desc = reading manifest v1.7.0 in ghcr.io/interlynk-io/sbomqs: unknown:
 resource not found: repo ghcr.io/interlynk-io/sbomqs, tag v1.7.0 not found. 
Check https://docs.gitlab.com/runner/shells/index.html#shell-profile-loading for more information
```

**Comment by riteshnoronha** [comment]
@Hannessi yes this is accurate in v0.1.7 we did not support ARM. Thats the support we added to v0.1.8. Let me fix v0.1.8 so u can use it.

**Comment by riteshnoronha** [comment]
Closing as a known issue of 0.1.7, has been fixed in v0.1.9
